content/05.administrative.md

```diff
@@ -19,6 +19,5 @@ CSG's spouse was an employee of Alex's Lemonade Stand Foundation, which was a sp
 JAS, CLS, CJB, and JNT are or were employees of Alex's Lemonade Stand Foundation, a sponsor of this research.
 
 ## Figure Titles and Legends
-
-
-## Tables with Titles and Legends
+**Figure 1. Overview of the OpenPBTA Project.**
```

Comment:
Is this intentionally duplicated? I'd have the same suggestions as above, but not sure if this is really supposed to be in two places.

Reply:
commented below, but also removed the legend info until later